elements/pfe-cta/src/pfe-cta.js

```diff
@@ -13,42 +13,94 @@ class PfeCta extends PFElement {
     return "pfe-cta.html";
   }
 
+  get defaultStyle() {
+    return this.hasAttribute("priority") ? false : true;
+  }
+
+  static get observedAttributes() {
+    return [
+      "pfe-priority",
+      "pfe-color",
+      "pfe-alt",
+      "on"
```

Comment:
we need to remove alt & on @chrisdo1

Reply:
@starryeyez024 I actually had removed both of those but had not committed yet. Changed `pfe-color` to `color` as well.